PORTAL/jobs/_job.py

```diff
@@ -355,13 +355,13 @@ def _build_send_script(
 
             # Trigger the next job.
             pushd {_common.SYS_PATH_ENTRY}
-            {sys.executable} -u -m jobs internal-run-next -v --config {cfgfile} --logfile {queue_log} &
+            {sys.executable} -u -m jobs internal-run-next -v --config {cfgfile} --logfile {queue_log} {reqid.workerid} &
```

Comment:
It may be worth decoupling "queueid" from "workerid" here.  It doesn't matter currently, but it could later and it would be tricky to identify this use at that point.  The easiest way is to put `queueid = reqid.workerid` above and then use it here.

Alternately, we could add `Job.queueid` to track which queue the job came from (and populate it for now with `reqid.workerid`).  However, that might not pay for itself and the above simple solution is enough for now.

This is only something to consider.  I'm also fine with leaving it the way you have it if you don't think the alternative is worth it.